source/fundamentals/crud/read-operations/change-streams.txt

```diff
@@ -288,3 +288,56 @@ output:
 
 For a list of options, see the `FullDocument <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/changestream/FullDocument.html>`__
 API documentation.
+
+Split Large Change Stream Events
+--------------------------------
+
+Starting in MongoDB 7.0, you can use a ``$changeStreamSplitLargeEvent``
```

Comment:
Suggestion:
Use the specific article "the" since only one of them exists.
```suggestion
Starting in MongoDB 7.0, you can use the ``$changeStreamSplitLargeEvent``
```